internal/wazeroir/format.go

```diff
@@ -6,7 +6,7 @@ import (
 
 const EntrypointLabel = ".entrypoint"
 
-func Format(ops []Operation) string {
+func Format(ops []UnionOperation) string {
 	buf := bytes.NewBuffer(nil)
 
 	_, _ = buf.WriteString(EntrypointLabel + "\n")
```

Comment:
the for loop below results in copy now as `ops` is the slice of values, not pointers. could you make it not copy? (refer somewhere else in the code base where we do the same thing)

Reply:
I think you mean `for i := range ops { op:=&ops[i] }` right?